public/views/create.html

```diff
@@ -87,12 +100,22 @@
             <i ng-if="!hideAdv" class="icon-arrow-up4"></i>
           </a>
         </div>
-        <div ng-hide="hideAdv" class="row">
+        <div ng-hide="hideAdv" class="row" ng-init="hideSlots=true">
           <div class="large-12 columns">
             <label for="network-name" class="line-b oh">
               <span translate>Testnet</span>
               <switch id="network-name" name="isTestnet" ng-model="isTestnet" class="green right m5t m10b"></switch>
             </label>
+            <label ng-show="create.isChromeApp()" for="hw-ledger" class="line-b oh">
+              <span translate>Use Ledger hardware wallet</span>
+              <switch id="hw-ledger" name="hwLedger" ng-model="hwLedger" class="green right m5t m10b" ng-change="hideSlots=false"></switch>
```

Comment:
If I do `ng-change="hideSlots=!hideSlots"` instead of `ng-change="hideSlots=false"` I have a strange JS error at runtime. So right now if you switch off the Ledger HW it won't hide the select. Any idea?